python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -883,6 +931,8 @@ def startup(self):
             self.repository.startup()
             # All Ids could have changed
             self.changed_ids = {}
+            self.flush_thread = RegistryFlusher(self)
+            self.flush_thread.start()
```

Comment:
Ok, I will add a configuration variable to change the sleep time between flushes as well as turn it off completely.